web-client/src/components/SignUpForm.jsx

```diff
@@ -0,0 +1,354 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Button from "@material-ui/core/Button";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import Container from "@material-ui/core/Container";
+import Visibility from "@material-ui/icons/Visibility";
+import VisibilityOff from "@material-ui/icons/VisibilityOff";
+import IconButton from "@material-ui/core/IconButton";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import * as yup from "yup";
```

Comment:
Are you sure that we need all methods from "yup"?

Reply:
I'll check it

Reply:
I read that from [https://rossbulat.medium.com/introduction-to-yup-object-validation-in-react-9863af93dc0e]
> "I recommend importing the entire library as you start developing, saving you the hassle of revisiting your imports every time you introduce a new Yup object as your forms evolve. When your app is ready to be built, revisit Yup to check whether you are only using a limited amount of components — perhaps you are only utilising the required and string objects — in this case we can simply import those 2 things:
> `import { string, required } from 'yup';`
> Overall though, if you are utilising more than around 5 components, it can become a code smell to be importing this many objects."
so I think we should import all methods